src/pybind/mgr/orchestrator/module.py

```diff
@@ -1327,6 +1327,14 @@ def _upgrade_check(self,
         raise_if_exception(completion)
         return HandleCommandResult(stdout=completion.result_str())
 
+    @_cli_read_command('orch upgrade ls')
+    def _upgrade_ls(self) -> HandleCommandResult:
```

Comment:
Would there be any benefit in allowing users to provide which registry to check here? For example, a user could provide "quay.io" and "ceph/ceph" and we'd check there and we have the info from the current image as the default. I'm trying to think if there's any case where somebody would be on an image from a different repo than the one they'd like to upgrade to or just check what's available on some other repo.